### DISCUSSION TITLE
Representation of different vegetation types with LAI

### DISCUSSION THREAD

**Discussion by calvarezgarreton** [mhm-ufz/mHM#8] [Q&A] [answered]
Hi, 

I am implementing mHM in order to disentangle the effect of climate and land cover on water availability, using the CAMELS-Cl dataset in Chile. 

I know the model only takes 3 land cover classes, which define some of the snow and soil moisture parameters. Given that we need to simulate ~10 vegetational classes, including native forest and forest plantations, both classified as 'lc=forest', we thought on characterising these vegetation types by their LAI phenology (gridded mean monthly values).

Do you think the LAI specification is enough to characterise ET dependency on the different vegetation types? Could you please indicate how lai.nc is taken into account by the model? In the equations from Samaniego et al., 2010, the only parameter depending on LAI is beta1, which affects only the canopy interception. Is this the only process depending on LAI or are there any other transferring function parameter (and thus other processes) also considering LAI? 

All the best,
Camila

**Comment by lese66** [comment, marked as answer]
Hi Camila,

mHM was/is an ongoing project that never ends. We started with three LC classes because that is what I could have in for my dissertation bach in 1998-2002. We did not have Sentinel data in 2006, when mHM started. If I would start again, now, I would include more LC classes. This is a ToDo for us... 
LAI was also not available for large regions and time spans and hence, I introduced monthly climatologies. Later we created a possibility to read LAI when MODIS data and other sources became available. There is a PhD at UFZ (Bahar) that is developing a dynamic LAI model for mHM. In the EDgE project, we used distributed LAI climatologies for every cell, and month. 

Unfortunately, Samaniego et al 2010 is not anymore a good reference for the equations available for mHM. Check the user manual. Originally yes, LAI only for canopy interception. This paper is, however, the basis for MPR.
Currently, LAI appears in several equations, e.g. PET corrections (M.Cuneyd Demirel & Simon Stisen Jun 2017), root depth. Check with  grep LAI ./mHM/*.f90. The PET LAI correction affects ET and SM. see https://hess.copernicus.org/articles/22/1299/2018/

We need to improve the vegetation dynamics in mHM (not as complex as in CLM) but still process based. This is crucial for improving ET estimation in semi-arid regions. One possibility we tried is to use Penman-M ET parameterization and use LAI to estimate aerodynamic resistance.
In the default mHM, you are right. LAI only affects interception using the simplest parameterization we have. Here new options can/should be tested. 

I hope it helps,

Luis

**Reply by calvarezgarreton** [reply]
Thanks for the complete answer.
I'll try the PET-LAI correction and test sensitivity to the different LAI climatologies.
All the best,
Camila
